### DISCUSSION TITLE
gradle and embedded native artifacts

### DISCUSSION THREAD

**Discussion by basix86** [jcefmaven/jcefmaven#51] [Q&A] [answered]
Hi.
Can you help me to configure a Gradle project with embedded native artifacts, on Windows?

in my build.gradle I have added the following lines

```
implementation 'me.friwi:jcefmaven:100.0.14.2'
implementation 'me.friwi:jcef-natives-windows-amd64:jcef-0aee584+cef-100.0.14+g4e5ba66+chromium-100.0.4896.75'
```

Now when I build the project I found the files:
```
jcef-api-jcef-0aee584+cef-100.0.14+g4e5ba66+chromium-100.0.4896.75.jar
jcefmaven-100.0.14.2.jar
jcef-natives-windows-amd64-jcef-0aee584+cef-100.0.14+g4e5ba66+chromium-100.0.4896.75.jar
```

Now the method (from CefAppBuilder:185)
```
InputStream nativesIn = PackageClasspathStreamer.streamNatives(
                    CefBuildInfo.fromClasspath(), EnumPlatform.getCurrentPlatform());
```
return always false and the natives are downloads.

How I can prevent the download?
Thanks!

**Comment by basix86** [comment, marked as answer]
After a clean project, it works correctly. You can close the discussion

**Reply by FriwiDev** [reply]
I'm happy to hear that :)
